Add chain link msg docs (#592)
desmos-labs/desmos: docs/developers/msgs/profiles/link-chain-account.md

# `MsgLinkChainAccount`
This message allows you to link an external chain address 
and add the [chain link](../../types/profiles/chain-link.md) of it to your Desmos profile.

## Structure

```json
{
    "@type":"desmos/MsgLinkChainAccount",
    "chain_address": {
        "@type": "/desmos.profiles.v1beta1.Bech32Address",
        "prefix": "<Bech32 prefix of the external chain account>",
        "value": "<Address of the external chain account>"
    },
    "chain_config": {
        "name": "<Name of the target external chain>"
    },
    "proof": {
        "pub_key": {
            "@type": "/cosmos.crypto.secp256k1.PubKey",
            "value": "<Base64 encoded public key>"
        },

>         "signature": "<Hex encoded signature created with private key associated given public key>",

Reviewer: ```suggestion
        "signature": "<Hex encoded signature created with private key associated to the given public key>",
```

>         "plain_text": "<Text for getting signature by signing with private key>"

Reviewer: ```suggestion
        "plain_text": "<Text signed with private key>"
```

    },

>     "signer": "<Desmos address of the profile witch link the chain account>"

Reviewer: ```suggestion
    "signer": "<Desmos address of the profile linking the chain account>"
```

}
```

### Attributes

| Attribute | Type | Description |
| :-------: | :----: | :-------- |
| `chain_address` | [AddressData](../../types/profiles/chain-link.md#Address) | Address data of the external chain account |
| `chain_config` | [ChainConfig](../../types/profiles/chain-link.md#ChainConfig) | Details of the target external chain |
| `proof` | [Proof](../../types/profiles/chain-link.md#Proof) | Data proving the ownership of the external chain account |

> | `signer` | String | Desmos address of the profile to which the link will be associated |

Reviewer: ```suggestion
| `signer` | String | Desmos address of the profile with which the link will be associated |
```
